packages/squiggle-lang/scripts/bench-sampleset-to-pointset.mjs

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+import { SqProject } from "@quri/squiggle-lang";
+import { measure } from "./lib.mjs";
+
+const maxP = 3;
+
+const sampleCount = process.env.SAMPLE_COUNT;
+
+for (let p = 0; p <= maxP; p++) {
```

Comment:
I imagine at some point in the future we won't need all files for each test? Like, this could just be a few lines of a larger "performance tests on various functions" file. Not a priority now though.

Reply:
Yeah, I'm gradually evolving it, allowing for common patterns to emerge.